ui/components/multichain/nft-item/nft-item.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import NftDefaultImage from '../../app/nft-default-image/nft-default-image';
+import {
+  AvatarNetwork,
+  BadgeWrapper,
+  BadgeWrapperAnchorElementShape,
+} from '../../component-library';
+import {
+  BackgroundColor,
+  Display,
+  Size,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box/box';
+
+export const NftItem = ({
+  alt,
+  name,
+  src,
+  networkName,
+  networkSrc,
+  tokenId,
+  onClick,
+  clickable = false,
+}) => {
+  return (
+    <Box
+      className="nft-item__container"
+      data-testid="nft-item"
+      as="button"
+      onClick={onClick}
+    >
+      <BadgeWrapper
+        className={classnames('nft-item__badge-wrapper', {
+          'nft-item__badge-wrapper__clickable': clickable,
+        })}
+        anchorElementShape={BadgeWrapperAnchorElementShape.circular}
+        positionObj={{ top: -4, right: -4 }}
+        display={Display.BLOCK}
+        badge={
+          <AvatarNetwork
+            className="nft-item__network-badge"
+            data-testid="nft-network-badge"
+            size={Size.SM}
+            name={networkName}
+            src={networkSrc}
+            borderWidth={2}
+            borderColor={BackgroundColor.backgroundDefault}
```

Comment:
```suggestion
            borderColor={BorderColor.borderDefault}
```

Reply:
Good catch @DDDDDanica, in most cases we wouldn't want to use the background colors with the border color prop but in this case because the the badge has this "cut out" type design where we want it too look as though the nft image has been cut out with the badge slightly smaller than the cut. We are intentionally breaking the color system here and using backgroundDefault for a border color. I think it's ok to use `BackgroundColor.backgroundDefault`

Reply:
Thanks for the background infor @georgewrmarshall . It makes sense to use `backgroundDefault` in this context, will resolve this one 🙏